index.html

```diff
@@ -102,9 +102,213 @@ <h2>1989</h2>
 			</div>
 		</div>
 	</div>
+	<!-- END OF STORY -->
+
+	<!-- QUOTE -->
+	<div class="container-fluid" id="review">
+		<div class="row text-center">
+			<div class="container">
+				<h1>Fresno Bee</h1>
+				<p>"The freshest and best ingredients come together to make some of the best food I have ever ate. You can really taste the Central Valley in every bite"</p>
+			</div>
+		</div>
+	</div>
+	<!-- END QUOTE -->
+
+<div class="container-fluid" id="menu">
+	<div class="container">
+		<div class="row text-center">
+			<div class="col-md-12">
+				<h1>Menu</h1>
+			</div>
+		</div>
+		<div class="row">
+			<div class="col-md-12">
+				<!-- Nav tabs -->
+			  <ul class="nav nav-tabs nav-justified" role="tablist">
+			    <li role="presentation" class="active"><a href="#appetizer" aria-controls="appetizer" role="tab" data-toggle="tab">Appetizer</a></li>
+			    <li role="presentation"><a href="#lunch" aria-controls="lunch" role="tab" data-toggle="tab">Lunch</a></li>
+			    <li role="presentation"><a href="#dinner" aria-controls="dinner" role="tab" data-toggle="tab">Dinner</a></li>
+			    <li role="presentation"><a href="#dessert" aria-controls="desert" role="tab" data-toggle="tab">Dessert</a></li>
+			  </ul>
+
+			  <!-- Tab panes -->
+				<div class="tab-content">
+			    <div role="tabpanel" class="tab-pane active" id="appetizer">
+			    	<div class="row food">
+			    		<div class="col-md-3">
+			    			<img src="img/dip.jpg" alt="" class="img-responsive img-thumbnail">
+								<h3>Appetizer 1</h3>
+								<p class="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deserunt perspiciatis veritatis sunt, obcaecati aut omnis, laborum provident amet possimus dicta!</p>
+			    		</div>
+			    		<div class="col-md-3">
+								<img src="img/eggplant.jpg" alt="" class="img-responsive img-thumbnail">
+								<h3>Appetizer 2</h3>
+								<p class="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deserunt perspiciatis veritatis sunt, obcaecati aut omnis, laborum provident amet possimus dicta!</p>
+			    		</div>
+			    		<div class="col-md-3">
+								<img src="img/meat.jpg" alt="" class="img-responsive img-thumbnail">
+								<h3>Appetizer 3</h3>
+								<p class="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deserunt perspiciatis veritatis sunt, obcaecati aut omnis, laborum provident amet possimus dicta!</p>
+			    		</div>
+			    		<div class="col-md-3">
+								<img src="img/bruschetta.jpg" alt="" class="img-responsive img-thumbnail">
+								<h3>Appetizer 4</h3>
+								<p class="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deserunt perspiciatis veritatis sunt, obcaecati aut omnis, laborum provident amet possimus dicta!</p>
+			    		</div>
+			    	</div>
+			    </div>
+					<div role="tabpanel" class="tab-pane fade" id="lunch">
```

Comment:
Is fade something she told us about adding in class?

Reply:
It was just a side mention when I asked Rocky about the effect.